internals/overlord/overlord.go

```diff
@@ -59,6 +59,8 @@ type Extension interface {
 type Options struct {
 	// PebbleDir is the path to the pebble directory. It must be provided.
 	PebbleDir string
+	// PebbleImportDirs is an optional ordered list of directory paths like PebbleDir to import data from.
+	PebbleImportDirs []string
```

Comment:
Naming bikeshed: I know we have the "Pebble" prefix on the `PebbleDir` field, but I think just `ImportDirs` would be fine here (like on daemon.Options).

Reply:
I've done this and also brought PebbleDir in line with other areas just referring to it as Dir.